src/core/turn-order.js

```diff
@@ -10,19 +10,19 @@
  * Standard move that simulates passing.
  *
  * Creates two objects in G:
- * passMap - A map from playerID -> boolean capturing passes.
+ * passOrder - An array of playerIDs capturing passes in the pass order.
  * allPassed - Set to true when all players have passed.
  */
 export const Pass = (G, ctx) => {
-  let passMap = {};
-  if (G.passMap !== undefined) {
-    passMap = { ...G.passMap };
+  let passOrder = [];
+  if (G.passOrder !== undefined) {
+    passOrder = G.passOrder;
   }
   const playerID =
     ctx.currentPlayer === 'any' ? ctx.playerID : ctx.currentPlayer;
-  passMap[playerID] = true;
-  G = { ...G, passMap };
-  if (Object.keys(passMap).length >= ctx.numPlayers) {
+  passOrder.push(playerID);
+  G = { ...G, passOrder };
+  if (passOrder.length >= ctx.numPlayers) {
```

Comment:
This should probably be changed to `passOrder.length >= ctx.playerOrder.length` so that it doesn't break when `playerOrder` contains duplicate entries.

Reply:
passOrder should be the same size of numPlayers, when all players passed